supersaintex/JudgeSnoopy
# modify "Integrate tensorflow"

## Description
I have revised the comment by Masahiro.

### Modifications
* CSS files can be loaded.
* The Prediction result can be seen on the web page.

### Test performed
* Go to the web page(http://localhost:8080/index)  after setting up the local web server and verify that the text has changed color.
* Confirm that the prediction result contains a number.
